images/create-env/create-env

```diff
@@ -76,7 +74,14 @@ set +u
 eval "$( conda shell.posix activate base )"
 set -u
 
-CONDA_YES=1 ${conda_impl} ${create_command} --prefix="${prefix}" "${@}"
+# Use CONDA_ALWAYS_COPY=1 to cut links to package cache.
+# (Which is esp. important if --strip or --remove-files are used!)
+CONDA_YES=1 \
+  CONDA_ALWAYS_COPY="${CONDA_ALWAYS_COPY:-1}" \
```

Comment:
Once could still run `CONDA_ALWAYS_COPY=0 create-env` in case copying is not wanted.